kratos/includes/gid_mesh_container.h

```diff
@@ -183,8 +183,8 @@ class GidMeshContainer
                                 || mGeometryType == GeometryData::Kratos_Line3D3 )
                         {
                             nodes_id[0] = (it)->GetGeometry() [0].Id();
-                            nodes_id[1] = (it)->GetGeometry() [2].Id();
-                            nodes_id[2] = (it)->GetGeometry() [1].Id();
+                            nodes_id[1] = (it)->GetGeometry() [1].Id();
+                            nodes_id[2] = (it)->GetGeometry() [2].Id();
```

Comment:
@josep-m-carbonell Ok but we have to be very careful to merge this in the Release. It can potentially break every other application results. I don't understand it well enough to know that, @RiccardoRossi can you take a second look at this?

Reply:
I have added the same correction in the #1684 for the master. Lines with 3 nodes in GiD and in Kratos are using the same convention, o--o--o  have local connectivity (1)---(3)---(2)

Reply:
There is no problem with the fix, I am just concerned that if other applications were expecting the old wrong order it may cause them to print something incorrect. I just want to be extra-careful with the release.

But again I am not sure if anyone else uses that, let me test the GiD examples that are using that before merging

Reply:
ok, thank you